app/src/main/java/com/example/encoratask/model/Character.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.encoratask.model
+
+import org.json.JSONObject
+import java.io.Serializable
+
+/**
+ * Created by Erick Sanchez
+ * Revision 1 - 09/06/21
+ */
+class Character:Serializable {
+
+    var id:Int = 0
+    var name:String = ""
+    var status = ""
+    var species = ""
+    var gender = ""
+    var image:String = ""
+
+    constructor(obj:JSONObject) {
+
+        this.id = obj.optInt("id")
+        this.name = obj.optString("name")
+        this.status = obj.optString("status")
+        this.species = obj.optString("species")
+        this.gender = obj.optString("gender")
+        this.image = obj.optString("image")
+    }
+
+    companion object {
+
+        fun parse(response:String) : ArrayList<Character> {
```

Comment:
This method must be moved to an extension method

Reply:
Good idea, but if you wanna use this in Java? it gonna be no friendly

Reply:
Look my comento above, there's a way to use extension functions in Java

Reply:
thanks bro